libraries/botbuilder/src/teamsActivityHelpers.ts

```diff
@@ -131,3 +131,19 @@ export function teamsNotifyUser(
         activity.channelData.notification = { alert: !alertInMeeting, alertInMeeting, externalResourceUrl };
     }
 }
+
+/**
+ * @param activity The current [Activity](xref:botframework-schema.Activity).
+ * @returns The current [Activity](xref:botframework-schema.Activity)'s team's onBehalfOf list, or null.
+ */
+export function teamsGetTeamOnBehalfOf(activity: Activity): OnBehalfOf[] {
+    validateActivity(activity);
+
+    const channelData = activity.channelData;
+    if (isTeamsChannelData(channelData)) {
+        const onBehalfOf = channelData.onBehalfOf;
+        return onBehalfOf || null;
+    }
+
+    return null;
```

Comment:
Looking at the rest of the functions in this class I'd change this to:
```suggestion
    if (isTeamsChannelData(activity.channelData)) {
        return activity.channelData.onBehalfOf || null;
    }
    return null;
```